tests/unit/ml_handlers/test_statsforecast.py

```diff
@@ -0,0 +1,185 @@
+import time
+from unittest.mock import patch
+import numpy as np
+import pandas as pd
+
+from mindsdb.integrations.handlers.statsforecast_handler.statsforecast_handler import StatsForecastHandler
+from statsforecast.models import AutoARIMA
+from statsforecast import StatsForecast
+from mindsdb_sql import parse_sql
+
+
+from tests.unit.executor_test_base import BaseExecutorTest
+
+
+def create_mock_df():
+    df2 = pd.DataFrame(pd.date_range(start="1/1/2010", periods=31, freq="Q"), columns=["time_col"])
+    df3 = df2.copy()
+
+    df2["target_col"] = range(1, 32)
+    df2["group_col"] = "a"
+    df2["group_col_2"] = "a2"
+    df2["group_col_3"] = "a3"
+
+    df3["target_col"] = range(11, 42)
+    df3["group_col"] = "b"
+    df3["group_col_2"] = "b2"
+    df3["group_col_3"] = "b3"
+
+    return pd.concat([df2, df3]).reset_index(drop=True)
+
+
+def test_statsforecast_df_transformations():
```

Comment:
Does this method run with the usual `pytest` call from the CI?

Reply:
It does indeed. Pytest will run all functions that start with "test_" or end with "_test"

Reply:
On a separate note, the CI currently skips tests for several of the ML handlers, only running Lightwood tests. 

I assume this is because of dependency conflicts etc.